src/Pages/ShopPage/ShopComponents/ProductCard/ProductCard.css

```diff
@@ -0,0 +1,18 @@
+.star-toggle-btn i {
+  color: var(--grey);
+}
+.star-toggle-btn i.filled {
+  color: var(--yellow-color);
+}
+.star-toggle-btn:hover {
+  background-color: transparent;
+}
+
+.btn.icon-btn.top-right {
+  top: 0;
+  right: 0;
+}
+
+.card:hover {
+  cursor: pointer !important;
```

Comment:
Hmm, Vikram suggested only use !important as last resort. If you didn't have any alternatives, then it is fine. Otherwise, you can try something else.

Reply:
not sure why i used it here. removed it.